README.md

```diff
@@ -1,72 +1,35 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Description the project.
+>a project to learn and apply to work with tools and professional behavior we are required to follow through the microverse program.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
```

Comment:
- [ ] Kindly keep the Getting started and Prerequisites section in your ReadMe, all headers that can be present should be present inside the ReadMe. You can add steps to clone the repo and run the live server in the Getting started section :)

You can use something like this, kindly personalize it for your Project


## Getting Started
#### Step 1:

- Clone this repo locally using git in the command line and the following command

  `git clone https://github.com/zhadier/My-Portofolio`
  
- Alternatively, you can just download the complete zip file and extract the folder in your directory

#### Step 2:
Copy All Project files and run using VSCODE. HTML can be viewed in browser.


### Prerequisites
- VSCODE

Reply:
Done, Thank you @zhadier .